.orchestra/ci/install-system-dependencies.sh

```diff
@@ -104,6 +104,14 @@ PACKAGES+=(ripgrep)
 apt-get -qq install --no-install-recommends --yes "${PACKAGES[@]}"
 
 if ! command -v git-lfs &> /dev/null; then
-  curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
-  apt-get -qq install --no-install-recommends --yes git-lfs
+  LFS_TMP=$(mktemp -d)
+  trap 'rm -rf $LFS_TMP' EXIT
+  LFS_URL=$(curl --silent https://api.github.com/repos/git-lfs/git-lfs/releases/latest | \
+            jq -r '.assets[].browser_download_url' | \
+            grep 'linux-amd64')
+  LFS_FILE=$(basename "$LFS_URL")
+  wget --quiet "$LFS_URL" -O "$LFS_TMP/$LFS_FILE"
+  tar -C "$LFS_TMP" --strip-components=1 -xf "$LFS_TMP/$LFS_FILE"
+  cp -a "$LFS_TMP/git-lfs" /usr/local/bin
+  git lfs install --skip-repo --system &> /dev/null
```

Comment:
I'd:
1. `curl -o- | tar xaf - ` so we avoid the temporary file
2. Extract only the file we need
3. Pin a git-lfs version and check the hash

Reply:
Ok, do note that this is different from the previous behavior where the latest version was fetched every time

Reply:
Done